packages/zendesk-adapter/src/filters/ticket_form.ts

```diff
@@ -154,12 +157,53 @@ const getChangeWithoutRemovedFields = (change: ModificationChange<InstanceElemen
   return clonedInst
 }
 
+const removeCustomTicketStatusFromTicketFieldIDsField = (
+  instance: InstanceElement,
+  customTicketElement: InstanceElement,
+): void => {
+  const ticketFieldIDs: Array<string | ReferenceExpression> = instance.value.ticket_field_ids || []
```

Comment:
are you sure its a string and not a number?
```suggestion
  const ticketFieldIDs: Array<number | ReferenceExpression> = instance.value.ticket_field_ids || []
```